app/chewy/library_index.rb

```diff
@@ -0,0 +1,19 @@
+class LibraryIndex < Chewy::Index
+  define_type LibraryEntry.includes(:media) do
+    root date_detection: false do
+      include IndexTranslatable
+
+      # Titles
+      translatable_field :titles, value: -> (e) { e.media.titles }
+      field :abbreviated_titles,
+        type: 'string',
+        value: -> (e) { e.media.abbreviated_titles }
+      field :status, type: 'string'
+      field :media_type, type: 'string'
+      field :progress, type: 'integer'
+      field :notes, type: 'string'
+      field :updated_at, type: 'date'
+      field :user_id, type: 'integer'
+    end
+  end
+end
```

Comment:
Would you mind removing this file?  I didn't mean to include it in the commit of badges :x My bad